docs/features/menu-legend-links.md

```diff
@@ -0,0 +1,39 @@
+# Menu/legend info Links
+gisida-react menu component builds menu items/sections based on json spec provided on clients site-config file. Menu/legend info links feature seeks to give context of the selected layer, menu sections and subsections. The links can be added by adding links and descriptions to layer/siteconfig as shown below.    
+
+## Usage and expected behaviour
+
+The following are configs to be added when implementing this feature.
+
+### Site level menu (sector/subsector)
+##### Sector level spec
+The `link` and `description` fields are optional one can be provided another ommitted as desired
+```
+"hyperLink": {
+        "Boundaries": {
+          "description": "desc",
+          "link": "https://gisida.com",
+        },
+        "BRCiS II Output Indicators": {
```

Comment:
I remember some keys here consisted of the parent menu name and the sub-menu menu name? Should we mention thus in the documentation?

Reply:
I think the parent layer should be added to the sub-menu and sub-categories only. Here is the doc that explains s that
https://github.com/onaio/gisida-react/pull/216/files#diff-e36307f8ee41ec0bb1f69c74262417f8639c5b6cf890dc7af5dbd74cc1592773R25-R33